validator/src/admin_rpc_service.rs

```diff
@@ -1417,6 +1438,14 @@ mod tests {
             actual_validator_id,
             expected_validator_id.pubkey().to_string()
         );
+        let event = votor_event_receiver
+            .recv()
+            .expect("Failed to receive SetIdentity event");
+        if let VotorEvent::SetIdentity = event {
```

Comment:
nit: can use
`assert_matches!(event, VotorEvent::SetIdentity)`
instead of the if else

Reply:
Err, tried that, got:
error[E0658]: use of unstable library feature `assert_matches`
    --> validator/src/admin_rpc_service.rs:1444:9
     |
1444 |         assert_matches!(event, VotorEvent::SetIdentity);
     |         ^^^^^^^^^^^^^^
     |
     = note: see issue #82775 <https://github.com/rust-lang/rust/issues/82775> for more information

So maybe we stick with the ugly if for now?